feature_extraction/fold_change.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Good looks. I should have thought to save them as well. Maybe we make a separate "results" folder. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/treyduffus/multiclass_lungcancer_classification_models/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
done adn done. moved all outputs to a dedicated results folder.